tests/test_parser.py

```diff
@@ -134,38 +119,144 @@ def test_parser_schema_violation():
         'very_long_metric_name_this_is_too_long_s_well' is too long
         """,
     ]
-
     # The validator reports a different error based on the python version, so
     # we need to provide two copies for this to work.
     if sys.version_info < (3, 7):
-        expected_errors.append(
-            """
+        expected_errors.extend(
```

Comment:
Now that we use OrderedDict for all Python versions, the error message changed slightly too